apns/abacustest/standalone.py

```
@@ -521,29 +540,16 @@
-    command_specified_here  = 'ulimit -c 0; '
-    command_specified_here += f'python3 {item}TestDriver.py -i driver.json | tee out.log'
-
-    jobdir, imag, cmd = init()
-
-    jobdir = folder_specified_here if jobdir is None else jobdir
-    imag = image_specified_here if imag is None else imag
-    cmd = command_specified_here if cmd is None else cmd
+    jobdir, imag, cmd = init(default_f=f'/path/to/folder',
+                             default_img='registry.dp.tech/dptech/abacus:3.8.4', # 3.8.3 is the version that FFT is fixed
+                             default_cmd='ulimit -c 0; ' 
+                                        + f'OMP_NUM_THREADS=1 mpirun -n 16 abacus | tee out.log')
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)